scripts/scrub-user.js

```diff
@@ -0,0 +1,195 @@
+const mysql = require('mysql');
+const config = require('../config.json');
```

Comment:
It IS, but if it didn't exist the rest of the site also wouldn't function (well the homepage would but you wouldn't be able to actually do anything). Having said that though, after the move to Kubernetes we'll be using env variables rather than config.json for most things so I'm going to refactor this pre-emptively anyway.